src/Rules/Install.hs

```diff
@@ -1,5 +1,5 @@
-module Rules.Install (installRules) where
-
+module Rules.Install () where
```

Comment:
Why comment this out? If the code cannot be compiled it will bitrot very quickly. I suggest to uncomment this and raise an issue discussing the future of install rules.

Reply:
This is another thing I told you about in my email, one of the things I wasn't quite sure what we should do about.

Reply:
I think I agree with @izgzhen on this: we should just remove installation rules and create an issue, explaining what needs to be brought back, pointing to the deleted code in the history.